examples/todo/data/database.js

```diff
@@ -34,12 +34,12 @@ class Database extends EventEmitter {
     [VIEWER_ID]: [],
   };
 
-  emit(topic, data) {
+  emit(channel, data) {
```

Comment:
just renaming variables here so we use `channel` everywhere

Reply:
"topic" is the more generic term in this context i believe... channel occasionally refers to some sort of more concrete implementation

e.g. see https://en.wikipedia.org/wiki/Publish–subscribe_pattern and naming in SNS, kafka, AMQP, and other systems. so i think the original naming here was probably better.

"channel" to refer to this in general looks almost specific to redis